docs/components/webapp.rst

```diff
@@ -43,6 +43,53 @@ Then you can start and stop the webapp, adjust files, and debug.
 FIXME(willkg): Review everything after this line.
 
 
+Setting up authentication and a superuser
+=========================================
+
+Creating a superuser
+--------------------
+
+If you want to do anything in the webapp admin, you'll need to create a
+superuser.
+
+Run this::
+
+  $ docker-compose run webapp python manage.py createsuperuser
+
+
+Then follow the prompts. The username and password don't matter at all, but the
+email address needs to match what you plan to use for Google auth.
+
+You can do this as many times as you like.
+
+
+Setting up the webapp for Google signin
```

Comment:
Nit: It's called Google Sign-In as that's a Google product name. We'd be more consistent if we call it this.